samples/MusicStore/Program.cs

```diff
@@ -15,8 +16,10 @@ public static void Main(string[] args)
                 .AddEnvironmentVariables(prefix: "ASPNETCORE_")
                 .Build();
 
+            var currentAssemblyLocation = new FileInfo(typeof(Program).GetTypeInfo().Assembly.Location);
+
             var builder = new WebHostBuilder()
-                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseContentRoot(currentAssemblyLocation.Directory.FullName)
```

Comment:
This isn't how we're supposed to use WebHostBuilder. It should be set to Directory.GetCurrentDirectory so that VS works correctly. What problem are you trying to resolve here?